Userland/Applications/Welcome/WelcomeWidget.cpp

```diff
@@ -79,7 +79,7 @@ ErrorOr<void> WelcomeWidget::create_widgets()
     };
 
     if (auto result = open_and_parse_tips_file(); result.is_error()) {
-        auto path = TRY(String::formatted("{}/tips.txt", Core::StandardPaths::documents_directory()));
+        auto path = "/res/tips.txt"_string;
```

Comment:
:man_shrugging: to me, the `/res` directory represents shared resources (i.e. data that can be used by multiple applications) and not a location where we just put everything that might possibly be picked up by more than one user.

The old location seemed fine, other systems use `/usr/share/doc` for shared documentation, or maybe someone comes up with a better suggestion?

Reply:
You are right the this file is not used by other applications. Maybe a better place would be: `/usr/share/Welcome/tips.txt` because it is a file the only the Welcome application uses and it is system wide. My main problem with the current location is that every user needs a copy of `tips.txt` in there `Documents` folder :| this would not scale with multiple users in theory.

Reply:
Well, `README.md` in the user root sort of has the same issue in that you'd need to copy it for every user, but we don't really do multiple users right now. `/usr/share/Welcome` would be fine with me as well.